File: ios/Plugin/CapacitorTapJackingPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(CapacitorTapJackingPlugin, "TapJacking",
           CAP_PLUGIN_METHOD(setFilterTouchesWhenObscured, CAPPluginReturnPromise);
)

File: ios/Plugin/CapacitorTapJackingPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(CapacitorTapJackingPlugin)
public class CapacitorTapJackingPlugin: CAPPlugin {
    private let implementation = CapacitorTapJacking()

    @objc func preventOverlays(_ call: CAPPluginCall) {        
        call.resolve();
    }

    @objc func enableOverlays(_ call: CAPPluginCall) {        
        call.resolve();
    }

}
